packages/rx_bloc_cli/mason_templates/bricks/feature_login/__brick__/lib/feature_login/ui_components/login_form.dart

```diff
@@ -71,20 +71,13 @@ class _LoginFormState extends State<LoginForm> {
             state: (bloc) => bloc.states.isLoading,
             builder: _buildLoginButton,
           ),
-          SizedBox(height: context.designSystem.spacing.xss1),
+          SizedBox(height: context.designSystem.spacing.xss1),{{#enable_forgotten_password}}
           TextButton(
-            onPressed: () => {{#enable_forgotten_password}}
-            GoRouter.of(context).push(PasswordResetRequestRoute().location),{{/enable_forgotten_password}}{{^enable_forgotten_password}}
-              showBlurredBottomSheet(
-                context: context,
-                builder: (BuildContext context) => MessagePanelWidget(
-                  message: context.l10n.featureLogin.notImplementedMessage,
-                  messageState: MessagePanelState.lessImportant,
-                ),
-              ),{{/enable_forgotten_password}}
+            onPressed: () =>
+              GoRouter.of(context).push(PasswordResetRequestRoute().location),
```

Comment:
Please prefer using the `go` method instead of `push` to ensure deep-link compatibility